### PR TITLE
[skip-ci] Add compute to groupby benchmarks

### PR DESCRIPTION
**Pull request by dcherian**
oops!

### CODE REVIEW

File: asv_bench/benchmarks/groupby.py [hunk 1]
```diff
@@ -69,7 +71,7 @@ def setup(self, *args, **kwargs):
 
         super().setup(**kwargs)
         self.ds1d = self.ds1d.to_dataframe()
-        self.ds1d_mean = self.ds1d.groupby("b").mean()
+        self.ds1d_mean = self.ds1d.groupby("b").mean().compute()
```
Comment (dcherian) [inline asv_bench/benchmarks/groupby.py:74]:
```suggestion
        self.ds1d_mean = self.ds1d.groupby("b").mean()
```

File: asv_bench/benchmarks/groupby.py [hunk 2]
```diff
@@ -88,7 +90,7 @@ def setup(self, *args, **kwargs):
         requires_dask()
         super().setup(**kwargs)
         self.ds1d = self.ds1d.chunk({"dim_0": 50}).to_dataframe()
-        self.ds1d_mean = self.ds1d.groupby("b").mean()
+        self.ds1d_mean = self.ds1d.groupby("b").mean().compute()
```
Comment (dcherian) [inline asv_bench/benchmarks/groupby.py:93]:
```suggestion
        self.ds1d_mean = self.ds1d.groupby("b").mean()
```

**Comment by Illviljan** [comment]
How come the benchmarks is not failing? Shouldn't it be significantly slower now?

**Comment by dcherian** [comment]
it runs the new benchmark on both `main` and `HEAD` so it should work as long as there is no error in the benchmarks

**Pull request merged**